BUILD.bazel

```diff
@@ -43,6 +44,25 @@ go_binary(
     x_defs = {"main.gitCommit": "{STABLE_GIT_COMMIT}"},
 )
 
+# Create the non-root user, "bazelremote", so the container can run with reduced
+# privileges.
+BAZEL_REMOTE_USER = "bazelremote"
+passwd_entry(
+    name = "bazelremote_passwd_entry",
+    username = BAZEL_REMOTE_USER,
+    info = BAZEL_REMOTE_USER,
+    uid = 1000,
+    gid = 1000,
+    home = "/data",
```

Comment:
I don't think we should reuse the cache directory as the home dir- bazel-remote might choose to delete unrecognised files in the cache dir, and this could become confusing if we forget and add some dot files or similar (eg a ~/.netrc file for the remote asset api).

Reply:
Done via a genrule that creates `/data/EMPTY` in a pkg_tar tarball (see comment in code for details)